extensions/cornerstone-dicom-sr/src/enums.js

```diff
@@ -0,0 +1,39 @@
+import { adaptersSR } from '@cornerstonejs/adapters';
+
+const { CodeScheme: Cornerstone3DCodeScheme } = adaptersSR.Cornerstone3D;
+
+export const CodeNameCodeSequenceValues = {
```

Comment:
It is definitely useful to have the enums provided here - could they be in one file per enum?  I generally find that easier to organize, but will let you decide if you want to do that.

Reply:
I believe having one file per enum is overkill (too many imports and files).